<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Общее</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="2">
       <widget class="QCheckBox" name="showDiagramms">
        <property name="text">
         <string>Диаграммы</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="showLabels">
        <property name="text">
         <string>Подписи</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="showLineouts">
        <property name="text">
         <string>Сноски</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Размер подписи</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="mLabelSizeSpinBox">
        <property name="suffix">
         <string> мм</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QTextEdit" name="editDiagrammStr"/>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="editTemplateStr"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>editDiagrammStr</sender>
   <signal>textChanged()</signal>
   <receiver>Form</receiver>
   <slot>on_editDiagrammStr_txt_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editTemplateStr</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>on_editTemplateStr_txt_changed(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_editTemplateStr_txt_changed(QString)</slot>
  <slot>on_editDiagrammStr_txt_changed()</slot>
 </slots>
</ui>
